include/render.h

```diff
@@ -3,6 +3,18 @@
 
 struct mako_state;
 
+enum mako_render_text_alignment {
```

Comment:
No need for this, pango already has it: https://developer.gnome.org/pango/stable/pango-Layout-Objects.html#pango-layout-set-alignment